charts/argo-workflows/values.yaml

```diff
@@ -12,6 +12,10 @@ init:
 
 createAggregateRoles: true
 
+# Override app name
+# Internally, this will be truncated at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+fullnameOverride: ""
+
```

Comment:
```suggestion
## String to partially override "argo-workflows.fullname" template
##
# nameOverride:

## String to fully override "argo-workflows.fullname" template
##
# fullnameOverride:
```

Reply:
Thank you for reviewing! I agree with it.

> Also nameOverride was missing. WDYT?

I completely forgot it... 😇 You are right.

I will fix as you suggested.

Reply:
I fixed on this commit 🙋 
https://github.com/argoproj/argo-helm/pull/782/commits/81e2e5d7708196997a5454394e0a86d11595c427